app/posts/smoking-cessation/2025/03/2025-03-20-smoking-cessation-pilot.md

```diff
@@ -0,0 +1,143 @@
+- shape the future service with user needs at its core
+
+We prepared the following research materials, so that when phase 1 begins, user research can take place as soon as possible.
+
+User surveys:
+- with NHS staff using the service (key insights into their experience)
+- with Tobacco Dependency Advisors (TDAs) who contribute to providing the service (their perspective and effectiveness of service)
+- where exemplar survey questions exist in the NHSE UCD Centre of Expertise, these have been re-used
+- these surveys will also act as a recruitment tool for qualitative research with both NHS staff and TDAs (additional recruitment methods will be available if needed)
+
+Discussion guides and consent forms:
+- discussion guides for in-depth interviews (to be amended as new insights emerge)
+- informed consent forms 
+
+### User survey
+
+The user survey is designed to gather insights from NHS staff who use the service, helping us understand their experiences, challenges, and the impact of the digital service on their quitting journey.
+
+The questions cover:
+- demographics and segmentation (age, gender, location, work pattern)
+- smoking behaviour (frequency, history, triggers)
+- motivation to quit (health conditions, attitudes)
+- digital readiness (comfort with tech, use of health apps)
+- overall experience and usability (service ease, impact on quitting, improvements)
+- journey through service:
+  - pre-service (finding and being introduced to the service) 
+  - accessing and registering for the service 
+  - quit attempt (users’ experience of actively trying to stop smoking with the service)
+  - discharge and exit from the service 
+  - returning or re-entering the service (where the previous quit attempt was unsuccessful)
+
+### Tobacco dependency advisor (TDA) survey
+
+The TDA survey focuses on the perspectives of those providing the service, capturing their views on the service’s effectiveness and usability. It also explores how the digital service compares to other cessation support methods and their experiences at different stages of the process:
+
+We ask TDAs about:
+- demographic information (age, location, experience)
+- overall assessment of the DTT service (usability, effectiveness, comparison with other services)
+- assessment of the journey through the service:
+- pre-service
+- access and registration
+- quit attempt
+- discharge and exit
+- re-entry 
+
+## What happened next?
+
+Unfortunately, this work has been indefinitely paused because of the latest spending review. This work is a critical element of preventative services as a reduction of smokers across England reduces future burden on NHS services. So, this work can be picked up again we packaged our findings into a [Confluence index](https://nhsd-confluence.digital.nhs.uk/display/PPP/Tobacco+Alpha) and developed a plan for future development. This allowed us to reflect on our progress and provide a foundation for any teams that may take on the work in the future.
```

Comment:
This might be more than one para? Consider splitting into short sharp ones. Not sure there needs to be a comma after "So, "?